examples/crd/cdk8s.yaml

```diff
@@ -3,4 +3,5 @@ language: typescript
 imports:
   - k8s@1.17.0
   - https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/crds/jenkins.io_jenkins_crd.yaml
-  - mattermost_crd.yaml
+  - moduleName: mattermost
+    file: mattermost_crd.yaml
```

Comment:
This somewhat acts as a test, but I'm wondering if we want more robust tests to cover this in `./tests/` or in `./packages/cdk8s-cli/test`?

Reply:
Yes. Definitely!

Reply:
Awesome, will do that then :)